lib/jekyll.rb

```diff
@@ -172,6 +172,7 @@ def sanitized_path(base_directory, questionable_path)
 require_all 'jekyll/commands'
 require_all 'jekyll/converters'
 require_all 'jekyll/converters/markdown'
+require_all 'jekyll/converters/smartypants'
```

Comment:
@parkr wat? I'm talking about the markdown and smartypants requires, both those belong in jekyll/coverters.rb.